frank/fit.py

```diff
@@ -241,12 +244,9 @@ def alter_data(u, v, vis, weights, model):
 
     wcorr_estimate = None
     if model['modify_data']['correct_weights']:
-        wcorr_estimate, weights = utilities.apply_correction_to_weights(u, v,
-                                                                        vis.real,
-                                                                        weights
-                                                                        )
+        weights = utilities.estimate_weights(u, v, vis, use_median=True)
 
-    return u, v, vis, weights, wcorr_estimate
+    return u, v, vis, weights
```

Comment:
Can you remove `wcorr_estimate` from the `Returns` in the docstring for `alter_data`?

Reply:
Done